domains/teamine.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "WellandDev",
+           "email": "rataretimig1502@gmail.com",
+           "discord": "905515659524472952"
+        },
+    
+        "record": {
+            "TXT": "domain for minecraft server dev."
```

Comment:
TXT record alone does nothing.

Reply:
Please forgive me, I was reading the brief documentation and rushed into action. 


Is it possible to re-register the subdomain?

Reply:
This is my first time working with domain names and I may be confused. I apologize in advance

Reply:
You can update the domain file on the fork of this repo on your account, and the changes will automatically be added to this PR